ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh

```diff
@@ -48,5 +48,6 @@ then
 fi
 
 openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-openshift-tests run-monitor --artifact-dir $STORE_PATH > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
+DISABLED_MONITOR_TESTS="apiserver-availability,apiserver-new-disruption-invariant,disruption-summary-serializer,external-service-availability,image-registry-availability,ingress-availability,pod-network-avalibility,service-type-load-balancer-availability"
+openshift-tests run-monitor --artifact-dir $STORE_PATH --disable-monitor=${DISABLED_MONITOR_TESTS} > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
```

Comment:
Is it possible to --enable-monitor to only run what we want? so we don't have to maintain this over time? Or do we need too many monitor tests.

Reply:
We will have a bigger list to include. We have total of 36 tests, only 8 to be excluded. 

Here is the list:

additional-events-collector, alert-summary-serializer, apiserver-availability, apiserver-new-disruption-invariant, audit-log-analyzer, cluster-info-serializer, clusteroperator-collector, disruption-summary-serializer, e2e-test-analyzer, etcd-log-analyzer, event-collector, external-service-availability, graceful-shutdown-analyzer, image-registry-availability, ingress-availability, interval-serializer, known-image-checker, kubelet-log-collector, legacy-authentication-invariants, legacy-cvo-invariants, legacy-etcd-invariants, legacy-kube-apiserver-invariants, legacy-networking-invariants, legacy-node-invariants, legacy-storage-invariants, legacy-test-framework-invariants, node-lifecycle, node-state-analyzer, operator-state-analyzer, pathological-event-analyzer, pod-lifecycle, pod-network-avalibility, service-type-load-balancer-availability, timeline-serializer, tracked-resources-serializer, upload-to-loki-serializer